akka-projection-grpc/src/main/scala/akka/projection/grpc/consumer/scaladsl/GrpcReadJournal.scala

```diff
@@ -144,6 +146,35 @@ object GrpcReadJournal {
       replicationSettings)
   }
 
+  /**
+   * INTERNAL API
+   *
+   * Factory method for replication, with replication specific serialization
+   */
+  @InternalApi private[akka] def apply(
+      settings: GrpcQuerySettings,
+      clientSettings: GrpcClientSettings,
+      replicationSettings: Option[ReplicationSettings[_]])(implicit system: ActorSystem[_]): GrpcReadJournal = {
+
+    // FIXME issue #702 This probably means that one GrpcReadJournal instance is created for each Projection instance,
+    // and therefore one grpc client for each. Is that fine or should the client be shared for same clientSettings?
+
+    val wireSerialization = new DelegateToAkkaSerialization(system)
```

Comment:
This apply is the same as the other apply above, aside from the serialization. Where is the other used? Do we need both? Is it even so that we can look at `replicationSettings.isDefined` to select serializer?

Reply:
Simplified to a single internal apply factory now.